setup.py

```diff
@@ -218,24 +218,24 @@ def write_version(filename: str = os.path.join(*["airflow", "git_version"])):
     'google-auth-httplib2>=0.0.1',
     'google-cloud-automl>=0.4.0',
     'google-cloud-bigquery-datatransfer>=0.4.0',
-    'google-cloud-bigtable==1.0.0',
+    'google-cloud-bigtable>=1.0.0',
     'google-cloud-container>=0.1.1',
-    'google-cloud-dataproc==0.5.0',
+    'google-cloud-dataproc>=0.5.0',
     'google-cloud-dlp>=0.11.0',
     'google-cloud-kms>=1.2.1',
     'google-cloud-language>=1.1.1',
-    'google-cloud-pubsub==1.0.0',
+    'google-cloud-pubsub>=1.0.0',
     'google-cloud-redis>=0.3.0',
     'google-cloud-spanner>=1.10.0',
     'google-cloud-speech>=0.36.3',
-    'google-cloud-storage~=1.16',
-    'google-cloud-tasks==1.2.1',
+    'google-cloud-storage>=1.16',
+    'google-cloud-tasks>=1.2.1',
     'google-cloud-texttospeech>=0.4.0',
     'google-cloud-translate>=1.5.0',
     'google-cloud-videointelligence>=1.7.0',
     'google-cloud-vision>=0.35.2',
     'grpcio-gcp>=0.2.2',
-    'httplib2~=0.9',
+    'httplib2>=0.9',
```

Comment:
This should probably stay as it is otherwise, when 1.0 of `httplib2` will be released, it might break things

'httplib2~=0.9' means 'httplib2>=0.9,==0.9.*'

Reply:
Hi @kaxil, the `~=` seems like a reasonable exception from time to time in both rolling and hard-pinned contexts. My conflict-free set of hard-pinned requirements actually has `httplib==0.15.0` so would you approve of upping this to `httplib2~=0.15`?

Reply:
Sorry @rayrrr It was a typo from my end.

`httplib2~=0.9` means `httplib2>=0.9,==0.*` (not `httplib2>=0.9,==0.9.*`)

So it would install the latest package that is `<1.0`

Reply:
Also, I am fine with upping it to `httplib2~=0.15` :)

Reply:
@kaxil done!